### ISSUE TITLE
Needs Documentation / Explanation of Loss, Accuracy and hits_at_k 

### ISSUE THREAD

**Issue by DougSilkstonekk** [uber/ludwig#131]
I understand that this might be second nature to some, but if you're releasing what is essentially being hyped as a codeless, black box toolkit, are there plans for a quick explanation on what some of these outputs actually mean?

Accuracy makes sense to me
Loss is not documented with an explanation or definition. High loss sounds bad though.
hits_at_k I can only assume is how many times out of one thousand, a hit occurs, whatever that is.

Forgive the ignorance, it's not for lack of trying.

**Comment by w4nderlust** [comment]
Hi @DougSilkstonekk . The tool is not "hyped" as code-free, it **IS** code free, it makes a lot of difference. Code-free **doesn't mean** machine-learning-knowledge-free.

Plus which loss is used for each feature is explicitly documented in the user guide. For instance by default category features have `softmax_cross_entropy` and [this](https://deepnotes.io/softmax-crossentropy) is the first result you find on google, which explains everything about it. Regarding `hits_at_k`, you can google that too and get [this](https://arxiv.org/pdf/1707.01476.pdf) as second result, in the last page there's the definition. S please put a minimal amount of effort and you'll find the answers.

We will always try to link to resources in the documentation that explain things, but our goal is to make a tool to make it easy to train models, not a tool for teaching you how to do machine learning and not a tool that doesn't require any machine learning knowledge to be used.
